### PR TITLE
Products page

### PR DESCRIPTION
**Pull request by ark2002**
- Added products page
- Linked products page from landing page

### CODE REVIEW

File: listingpage/listingpage.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
+.filter {
+    width: 21%;
+    min-width: 20rem;
+    height: 100%;
+    position: sticky;
+    top: 5rem;
+    left: 0;
+    padding: 3rem;
+    box-shadow: #0000004d 0px 19px 38px, #00000038 0px 15px 12px;
```
Comment (shrushti2000) [inline listingpage/listingpage.css:9]: try using CSS variables here .

File: listingpage/listingpage.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+.filter__top {
+    align-items: center;
+    gap: 12rem;
+}
+
+.productlist__container {
+    width: 77%;
+    margin-left: auto;
+    background-color: #f3f3f3;
```
Comment (shrushti2000) [inline listingpage/listingpage.css:22]: Here also use CSS variables.

File: listingpage/listingpage.css [hunk 3]
```diff
@@ -0,0 +1,77 @@
+.filter {
+    width: 21%;
```
Comment (abhishek17551) [inline listingpage/listingpage.css:2]: `width:23%`

File: listingpage/listingpage.css [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    margin-left: auto;
+    background-color: #f3f3f3;
+    position: relative;
+}
+
+.productlist {
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+}
```
Comment (abhishek17551) [inline listingpage/listingpage.css:30]: can be declared as class and reused across the project

**Review by abhishek17551 — APPROVED** [review]
Minute changes. Code  is clean, UI is responsive!

**Review by shrushti2000 — APPROVED** [review]
Overall everything looks good. Code structure is also good. Just a suggestion , if possible make the filter sidebar such that it  covers the entire vertical height of the screen.

**Comment by ark2002** [comment]
> Overall everything looks good. Code structure is also good. Just a suggestion , if possible make the filter sidebar such that it covers the entire vertical height of the screen.

it does just i dont want it to overflow my footer thats why its limited . if new peoducts are added the sidebar increases

File: listingpage/listingpage.css (hunk 2 above)
Comment (ark2002) [reply listingpage/listingpage.css:22]: these colors dont really gonna be used across the project they are one off colors so decided againts the variables for them

File: listingpage/listingpage.css (hunk 1 above)
Comment (ark2002) [reply listingpage/listingpage.css:9]: same with this , don't really wanna use variables for box shadows

**Pull request merged**